branchout-project

```diff
@@ -5,14 +5,14 @@ set -e
 
 function projectList() {
   if test $# -eq 0; then
-    sort "${PROJECTION_DIRECTORY}/${BRANCHOUT_PROJECTS}"
+    sort -u "${PROJECTION_DIRECTORY}/${BRANCHOUT_PROJECTS}"
```

Comment:
I think this needs to be `LANG=C sort` rather than sort -u, otherwise you still get differences across platforms and across people using different locales

Reply:
that's an orthogonal concern to the point of the change, is it not?

Reply:
no you are changing the sort to be consistent across platform, but -u doesn't solve the problem, its just solves most of the cases

Reply:
oh -u, gotcha

Reply:
The title of the PR explains the PR...